src/components/Drawer.jsx

```diff
@@ -126,6 +127,8 @@ const SideMenuDrawer = React.memo(function SideMenuDrawer({ handleDrawerClose, d
 
   return (
     <SwipeableDrawer
+      onOpen={() => {}}
```

Comment:
why passing empty functions as prop instead of skipping the prop as a whole?

Reply:
These props are required props, in previous merge it was removed so was getting warnings in console, thats why added it again

Reply:
Ok cool